MANIFEST.in

```diff
@@ -11,15 +11,18 @@ include site.cfg.example
 include numpy/random/mtrand/generate_mtrand_c.py
 recursive-include numpy/random/mtrand *.pyx *.pxd
 # Add build support that should go in sdist, but not go in bdist/be installed
+# Note that sub-directories that don't have __init__ are apparently not
+# included by 'recursive-include', so list those separately
+recursive-include numpy
 recursive-include numpy/_build_utils *
-recursive-include numpy/linalg/lapack_lite *.c *.h
+recursive-include numpy/linalg/lapack_lite *
```

Comment:
I think this line should be reverted - excluded the intermediate files which are used only as a manual upgrade-lapack tool made sense to me.

Reply:
I think you're thinking that they're not needed for installation, hence don't package? I'm thinking of the sdist more as an archive of the tagged source tree (the first purpose in https://www.python.org/dev/peps/pep-0517/#terminology-and-goals). We also package `benchmarks`, `docs` and `tools`, and I don't see a very strong reason to just leave out these files.

Reply:
Correct, that was my line of reasoning. I buy your argument that these files are of roughly the same relevance as "tools" though.